### PR TITLE
[BUG][Document] Fix the bug that failed to build the help module

### PR DESCRIPTION
**Pull request by xy720**
## Proposed changes

There are multiple entries with same key in help documents, which will cause build help module failed.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
Better change the title to 
`show database id`, `show table id` and `show partition id`, for both Chinese and English doc.

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW PARTITION.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# SHOW DATABASE
+# SHOW PARTITION
```
Comment (EmmyMiao87) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW PARTITION.md:27]: please change English document

Comment (xy720) [reply docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW PARTITION.md]: changed it.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**